divviup/src/main/java/org/divviup/android/Client.java

```diff
@@ -155,6 +307,6 @@ public HPKEConfigList(byte[] bytes) {
     }
 
     private enum VDAF {
```

Comment:
style nit: sorry, missed this in the previous PR -- Java style is generally to capitalize the first letter of each "word" in the type name, even if one of the words is an initialism. that is, this should be `Vdaf` rather than `VDAF`.

Reply:
Fixed this with `HpkeConfigList` as well